calc.html

```diff
@@ -1,91 +1,82 @@
+                document.forms.inputs.result.value = document.forms.inputs.number1.value - document.forms.inputs.number2.value;
+            },
+            division: function() {
+                // @TODO: complete this code so it returns the correct result
+                document.forms.inputs.result.value = document.forms.inputs.number1.value - document.forms.inputs.number2.value;
             }
         }
     </script>
     <style>
         body {
-            padding: 30px;
+            padding: 40px;
         }
+        
         .result {
-            padding-top: 20px;
+            padding-top: 30px;
         }
     </style>
 </head>
+
 <body>
     <!-- https://getbootstrap.com/docs/4.0/components/card/ -->
-    <div class="card" style="width: 28rem;">
+    <div class="card" style="width: 26rem;">
         Calculator
         <div class="card-body">
             <form name="inputs">
                 <div class="container">
                     <div class="form-group">
                         <label>Number 1:</label>
-                        <input
-                            type="number"
-                            name="number1"
-                            class="form-control"
-                            placeholder="Number 1"
-                        >
+                        <input type="number" name="number1" class="form-control" placeholder="Number 1">
                         <small class="form-text text-muted">Enter the first number</small>
                     </div>
                     <div class="form-group">
                         <label>Number 2:</label>
-                        <input
-                            type="number"
-                            name="number2"
-                            class="form-control"
-                            placeholder="Number 2"
-                        >
+                        <input type="number" name="number2" class="form-control" placeholder="Number 2">
                         <small class="form-text text-muted">Enter the second number</small>
                     </div>
                     <div aria-label="Buttons">
-                        <button type="button" class="btn btn-primary" onclick="calculator.add()">add</button>
-                        <button type="button" class="btn btn-primary" onclick="calculator.subtract()">subtract</button>
-                        <button type="button" class="btn btn-secondary" onclick="calculator.multiply()">multiply</button>
+                        <button type="button" class="btn btn-primary" onclick="calculator. + ()">+</button>
+
+                        <button type="button" class="btn btn-primary" onclick="calculator. - ()">-</button>
+
+                        <button type="button" class="btn btn-primary" onclick="calculator. * ()">*</button>
+
+                        <button type="button" class="btn btn-primary" onclick="calculator. / ()">/</button>
```

Comment:
Your initial implementation of the functions in the format of "calculator.add()" was correct and would have worked properly, however your replacement won't work as the syntax of "calculator. + ()" is not syntactically correct and therefore won't be recognized or run. If you need additional help with this or further explanation please don't hesitate to schedule some time with me via my calendly.